### ISSUE TITLE
About the accuracy on UCF101

### ISSUE THREAD

**Issue by zhengying1013** [hbilen/dynamic-image-nets#9]
Hi, I find a strange problem when evaluating your code on ucf101. The accuracies are shown below.
Accuracy on split 1: 0.659001
Accuracy on split 2: 0.996249
Accuracy on split 3: 0.997294

Do you have any idea about the huge difference?

**Comment by hbilen** [comment]
Did you set opts.split to the right split while testing?

**Comment by zhengying1013** [comment]
Thanks for your reply. I have carefully checked the options and compared the predicted labels with ground truth. The problem still exists. If that is convenient, can you tell me the accuracies on the three splits?

**Comment by zhengying1013** [comment]
I have found the reason for this problem. As only the model trained on split1 provided, I use the model to test videos of split2 and split3. Because most of the test videos are already exist in the training set of split1, the accuracies are very high.
